src/vmm_mad/remotes/kvm/poll

```diff
@@ -157,8 +158,7 @@ module KVM
             end
 
             if (values[:state] == 'a') && (vm[:reason] != 'migrating')
-                values.merge!(get_interface_statistics(name, xml))
-                values.merge!(get_diskio_statistics(name, xml))
+                values.merge!(get_io_statistics(name, xml))
```

Comment:
1. Similar change should be done in method `def self.get_vm_info(one_vm)`.
2. Methods `get_intertface_statics` and `get_diskio_statistics should` should be removed as they provide redundant functionality.